src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1416,19 +1419,22 @@ public override BoundNode VisitCall(BoundCall node)
             return null;
         }
 
-        private static bool HasExplicitTypeArguments(BoundCall node)
+        private static bool HasImplicitTypeArguments(BoundCall node)
```

Comment:
>HasImplicitTypeArguments [](start = 28, length = 24)

I think we should record the fact in the node and not try to infer things from the syntax. We can have scenarios when syntax is not an invocation syntax, for example for a call that is synthesized as part of a query interpretation.

Reply:
Added a `// PROTOTYPE` comment for now.

---
In reply to: [169828224](https://github.com/dotnet/roslyn/pull/24935#discussion_r169828224) [](ancestors = 169828224)